BusinessServices/PlayerManager.cs

```diff
@@ -33,9 +40,8 @@ public Player AddPlayer(PlayerHandshake info)
             var newPlayer = new Player(info,
                 WorldState.Instance.GetRandomGridCell(),
                 //WorldGrid[0, 0],
-                new Vector2(GlobalConfigs.GridCellWidth / 2, GlobalConfigs.GridCellHeight / 2));
-
-
+                new Vector2(GlobalConfigs.GridCellWidth / 2, GlobalConfigs.GridCellHeight / 2),
+                _playerColors[info.Name.Min() % _playerColors.Length]);
```

Comment:
Didn't feel like getting a random number

Reply:
ooooh, that's a clever way to do that!